tests/page-objects/forms/immunization-visit-form.wdio.page.js

```diff
@@ -0,0 +1,93 @@
+let countAppliedVaccines = 0;
```

Comment:
I'm not 100% convinced of keeping a data state here, can you move it to `immunization-visit-form.standard-wdio-spec.js` just after `let babyMedicID = '';`? 

I think page objects are helpers to get and set DOM elements. Once the vaccine is applied then the page object tells the spec that it was applied and the spec keeps record or it.